charts/ceph-csi-cephfs/templates/nodeplugin-daemonset.yaml

```diff
@@ -37,6 +37,10 @@ spec:
       # to use e.g. Rook orchestrated cluster, and mons' FQDN is
       # resolved through k8s service, set dns policy to cluster first
       dnsPolicy: ClusterFirstWithHostNet
+{{- if .Values.nodeplugin.imagePullSecrets }}
+      imagePullSecrets:
+{{ toYaml .Values.nodeplugin.imagePullSecrets | indent 8 -}}
```

Comment:
Could you show in an example what value you would pass? I would expect something like this instead:

```yaml
 - name: {{ .Values.nodeplugin.imagePullSecrets }}
```

Reply:
Here you should provide an array like:

```yaml
imagePullSecrets:
  - name: my-secret-a
  - name: my-secret-b
```

This part in fact is copied from csi-driver-nfs:

https://github.com/kubernetes-csi/csi-driver-nfs/blob/862cb036e3f0d7c3fe28d2b127a6f96335b15309/charts/v4.3.0/csi-driver-nfs/values.yaml#L127-L131

https://github.com/kubernetes-csi/csi-driver-nfs/blob/862cb036e3f0d7c3fe28d2b127a6f96335b15309/charts/v4.3.0/csi-driver-nfs/templates/csi-nfs-controller.yaml#L20-L23

And It is a recommended way rather than: 

```
imagePullSecrets:
  - name: {{ .Values.nodeplugin.imagePullSecret }}
```

In this way you can not provide multiple secrets. And maybe there are other fields (no just `name` only).

Reply:
ok, thanks for the clarification!